python/ray/gateway.py

```diff
@@ -0,0 +1,112 @@
+import click
+
+# TODO (dsuo): eventually remove this dependency
+from flask import Flask, request, send_file
+import io
+import pyarrow
+from pyarrow import plasma as plasma
+
+from ray.utils import hex_to_binary
+
+app = Flask(__name__)
+ctx = pyarrow.default_serialization_context()
```

Comment:
It's possible that we already discussed this and I forgot the relevant issues, but I think it should be possible to get away without having a serialization context in the gateway (e.g., to never have the gateway serialize or deserialize anything).

Reply:
Yep. This was just a copy over; haven't gotten to it yet.